Cargo.toml

```diff
@@ -26,3 +27,6 @@ name = "tree-sitter-grep"
 
 [workspace]
 members = ["tree_sitter_grep_filter_before_line_number"]
+
+# [profile.release]
```

Comment:
I just had this set per the [`cargo-instruments`](https://crates.io/crates/cargo-instruments) docs, could remove